openadapt/signals.py

```diff
@@ -95,16 +97,16 @@ def __setup_file_signal(self, file_path):
         except pd.errors.ParserError:
             logger.info(f"Error: Pandas could not parse the excel file, possible formatting issue.")
         return description
-        
 
 
     def __setup_url_signal(self, http_url):
         """
         Read a description of a signal from an HTTP URL.
         """
         # Get the signal from the URL.
-        #headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3'}
-        response = requests.get(http_url, allow_redirects=True)
+        headers = {'User-Agent': 'Mozilla/5.0 (iPad; CPU OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148'}
```

Comment:
Please break into multiple lines 🙏

Reply:
Resolved in [0adf43d](https://github.com/MLDSAI/OpenAdapt/pull/160/commits/0adf43d4a585ba2cf5d3850c9856a0af5646ab58)